examples/terraform-submodules/gke/README.md

```diff
@@ -0,0 +1,33 @@
+This terraform module creates an Agones cluster on GKE. 
+
+# Prerequisites
+- Terraform 0.12.x
+- Helm 3.x
+- An existing GCP Project
+- Enabled service API(s)
+
+# Enabling Service APIs
+
+Enable each required API below by following [the enabling APIs guide](https://cloud.google.com/endpoints/docs/openapi/enable-api#gcloud).
+
+Required APIs:
+- container.googleapis.com (Kubernetes)
+
+# Examples
+
+## Create an Agones in a default VPC.
+
+
+Required fields:
+- project
+
+```terraform apply -var project="<YOUR_GCP_ProjectID>" [-var agones_version="1.6.0"]```
+
+## Create an Agones cluster in a custom VPC.
+Required fields:
+- project
+- network
```

Comment:
Let use default value "default" for network if not defined
```suggestion
```
similar to:
```
  network           = lookup(var.cluster, "network", "default")
```
And probably add the code to omit subnetwork also.